app/Filament/Resources/ReferenceResource.php

```diff
@@ -34,14 +34,17 @@ public static function form(Form $form): Form
                             ->label("Imię i nazwisko autora")
                             ->required()
                             ->maxLength(255),
+                        Forms\Components\Radio::make("sex")
+                            ->label("Płeć")
+                            ->required()
+                            ->options([
+                                "male" => "Mężczyzna",
+                                "female" => "Kobieta",
```

Comment:
```suggestion
                                "male" => "Mężczyzna",
                                "female" => "Kobieta",
```
What about other options like "nie chcę podawać" or smth like that.

Reply:
Do we really need that? It's just an option to make it easier to choose a placeholder image - we have male and female placeholders.